docs/overview/concepts/indexing.md

```diff
@@ -7,11 +7,11 @@ sidebar_position: 1
 
 Quickwit ingests JSON records and refers to them as "documents" or "docs". Each document must be a JSON object. When ingesting files, documents must be separated by a newline.
 
-As of version 0.2, Quickwit does not support file formats such as `Avro` or `CSV`. Compression formats such as `bzip2` or `gzip` are also not supported.
```

Comment:
As a user what I understand with `As of version 0.2,`  is that it was previously supported and has been removed or changed.